### PR TITLE
CPPReference Fathead: Update README file

### PR DESCRIPTION
**Pull request by rasikapohankar**
## Description of new Instant Answer, or changes

Updates the README file to indicate where the parsing script is located.

## Related Issues and Discussions

Related to the discussion in #407 and #539 

## People to notify

@moollaza @sahildua2305 

---

Instant Answer Page: https://duck.co/ia/view/cppreference_doc
Maintainer: @ManrajGrover

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/fathead/cppreference_doc/README.md [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+Data Source
 -----------
 
 fetch.sh downloads and untars a snapshot of the website, which is available at
-http://en.cppreference.com/w/Cppreference:Archives. A new snapshot is released
-periodically. The location of the newest archive that is tested for DuckDuckGo
-is upload.cppreference.com/w/Cppreference:DDG-link?action=raw.
+http://en.cppreference.com/w/Cppreference:Archives. A new snapshot is released periodically.
+
+The location of the newest archive that is tested for DuckDuckGo is upload.cppreference.com/w/Cppreference:DDG-link?action=raw.
```
Comment (sahildua2305) [inline lib/fathead/cppreference_doc/README.md]: @rasikapohankar let's hyperlink this URL as well. Starting the URL with `http://` will hyperlink it in markdown.

File: lib/fathead/cppreference_doc/README.md [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
+This archive version maybe different from the latest public archive present at http://en.cppreference.com/w/Cppreference:Archives.
+
+The archive at upload.cppreference.com/w/Cppreference:DDG-link?action=raw is updated by linking to the most recent cppreference archive that someone actually tested to work, to ensure that the Fathead functionality does not accidentally break when new cppreference archives are released.
```
Comment (sahildua2305) [inline lib/fathead/cppreference_doc/README.md]: Same here as well.

File: lib/fathead/cppreference_doc/README.md (hunk 1 above)
Comment (rasikapohankar) [inline lib/fathead/cppreference_doc/README.md]: Done

**Pull request merged**